src/tracer.js

```diff
@@ -70,6 +75,11 @@ export default class Tracer {
         this._reporter = reporter;
         this._sampler = sampler;
         this._logger = options.logger || new NullLogger();
+        if (options.baggageRestrictionManager) {
```

Comment:
this could be simpler written as:
this._baggageSetter = new BaggageSetter(
    options.baggageRestrictionManager || new DefaultBaggageRestrictionManager(),
    this._metrics
);